custom/icds_reports/static/js/directives/mobile-datepicker/mobile-datepicker.directive.js

```diff
@@ -311,9 +311,20 @@ angular.module('icdsApp').directive('mobileDatepicker', function() {
 
 
             var today = $scope.date;
+            var months = ['January','February','March','April','May','June','July','August','September','October','November','December'];
+            let center = $scope.date.getMonth();
+            if ($scope.date.getFullYear() === 2017) {
+                center = $scope.date.getMonth()-2;
+            } else if ($scope.date.getFullYear() === new Date().getFullYear()) {
+                if ($scope.date.getMonth() > new Date().getMonth()) {
+                    center = new Date().getMonth();
+                }
+            }
             $('.month').WSlot({
-                items:['January','February','March','April','May','June','July','August','September','October','November','December'],
-                center:today.getMonth(),
+                items:($scope.date.getFullYear() === 2017) ? months.slice(2) :
```

Comment:
I think it'd be worth leaving a comment or two about how we manually override the months to match the project start date